easybuild/easyconfigs/n/NCCL/NCCL-2.18.3-GCCcore-12.3.0-CUDA-12.1.1.eb

```diff
@@ -11,8 +11,11 @@ toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
 github_account = 'NVIDIA'
 source_urls = [GITHUB_SOURCE]
 sources = ['v%(version)s-1.tar.gz']
-checksums = [('6477d83c9edbb34a0ebce6d751a1b32962bc6415d75d04972b676c6894ceaef9',
-              'b4f5d7d9eea2c12e32e7a06fe138b2cfc75969c6d5c473aa6f819a792db2fc96')]
+patches = ['NCCL-2.16.2_fix-cpuid.patch']
+checksums = [
+    {'v2.18.3-1.tar.gz': '6477d83c9edbb34a0ebce6d751a1b32962bc6415d75d04972b676c6894ceaef9'},
+    {'NCCL-2.16.2_fix-cpuid.patch': '0459ecadcd32b2a7a000a2ce4f675afba908b2c0afabafde585330ff4f83e277'},
+]
```

Comment:
Please restore the alternative checksum. See https://github.com/easybuilders/easybuild-easyconfigs/pull/18906

Reply:
Oh, didn't notice that. Done :+1:

Reply:
Hitting a framework bug. Shall we resolve that first or shall I revert to non-dicts?

Reply:
let's get this merged first?
checksumming is quite a mess at the moment. probably better to fix this in EB-5 so breaking changes can be made, see also https://github.com/easybuilders/easybuild-framework/issues/4346